app-libs/stf/src/lib.rs

```diff
@@ -40,8 +40,7 @@ pub type AuthorityId = <Signature as Verify>::Signer;
 pub type AccountId = AccountId32;
 pub type Hash = sp_core::H256;
 pub type BalanceTransferFn = ([u8; 2], AccountId, Compact<u128>);
-//FIXME: Is this really necessary to define all variables three times?
-pub static TEEREX_MODULE: u8 = 8u8;
+
 pub static UNSHIELD: u8 = 6u8;
```

Comment:
Could you also remove this unshield defintion? Should be imported from settings as well

Reply:
Ah well just noticed it's not even defined in  the settings.. @clangenb is there any reason UNSHIELD should not be defined in settings?

Reply:
It's only used in app-libs/stf. So I didn't move it

Reply:
Yes, but it's more userfriendly to have all pallet values defined in one place.

Reply:
Well, this file is a mess anyhow. So I don't really care. :) However, there is no reason to not put it there.

Reply:
Thank you for being strict :)

Reply:
Okay, then lets move it to settings. In case someone adapts the teerex-pallet and needs to find the scattered definitions in the code to adapt it... that's bad userexpierience.